JS/audio.js

```diff
@@ -0,0 +1,40 @@
+var audioObj = {
+    cheer1: 'audio/cheer1.wav',
+    cheer2: 'audio/cheer2.wav',
+    cheer3: 'audio/cheer3.wav',
+    cheer4: 'audio/cheer4.wav'
+}
+
+function listenKeypress() {
+    document.addEventListener('keypress', function(e) {
+        switch (true) {
```

Comment:
you could optimize this code using an object lookup:
```
var keys = {'q' : 'cheer1'} // etc
if (e.key in keys) {
    makeNoise(keys[e.key], audiObj)
}
```
```

Reply:
This shortened our code a lot, thanks. 👍